src/views/accounts/AccountPush/ExecutionDetail/AccountPushRecord.vue

```diff
@@ -57,6 +58,7 @@ export default {
             label: this.$t('Username'),
             formatter: DetailFormatter,
             formatterArgs: {
+              drawer: true,
               can: this.$hasPerm('accounts.view_account'),
               getTitle({ row }) {
                 return row.account.username
```

Comment:
The code provided is not included in its original context, so it's difficult to make specific comparisons or suggest improvements without that information. Could you please clarify which two sections of the codes need checking?